tools/tf2openapi/types/tfsavedmodel.go

```diff
@@ -18,25 +18,23 @@ type TFSavedModel struct {
 	MetaGraphs [] TFMetaGraph
 }
 
-func NewTFSavedModel(model *pb.SavedModel) (TFSavedModel, error) {
+func NewTFSavedModel(model *pb.SavedModel, sigDefKey string) (TFSavedModel, error) {
 	tfSavedModel := TFSavedModel{
 		MetaGraphs: []TFMetaGraph{},
 	}
 	for _, metaGraph := range model.MetaGraphs {
 		if !utils.Includes(metaGraph.MetaInfoDef.Tags, ServingMetaGraphTag) {
 			continue
 		}
-		tfMetaGraph, err := NewTFMetaGraph(metaGraph)
+		tfMetaGraph, err := NewTFMetaGraph(metaGraph, sigDefKey)
 		if err != nil {
 			return TFSavedModel{}, err
 		}
 		tfSavedModel.MetaGraphs = append(tfSavedModel.MetaGraphs, tfMetaGraph)
-
-	}
-	if len(tfSavedModel.MetaGraphs) == 0 {
-		return TFSavedModel{}, errors.New("model does not contain any servable MetaGraphs")
+		return tfSavedModel, nil
 	}
-	return tfSavedModel, nil
+	// len(tfSavedModel.MetaGraphs) is 0
```

Comment:
I think you could omit this comment since the error indicates what's happening. "len == 0" is not very expressive as a comment IMO.